eval/list_test.go

```diff
@@ -0,0 +1,133 @@
+package eval
+
+import "testing"
+
+func TestParseAndFixListIndex(t *testing.T) {
+	check := func(t *testing.T, name, value string, want, got interface{}) {
+		if want != got {
+			t.Errorf("%s value: [%s] want: [%v] got: [%v]",
+				name, value, want, got)
+		}
+	}
+
+	for _, item := range []struct {
```

Comment:
Can you move the definition of test data out of the function into a variable? Namely

```
var parseAndFixListIndexTests = ...
```